### PR TITLE
Fix nightly failures

### PR DESCRIPTION
**Pull request by ashwini-mhatre**
##### ISSUE TYPE

- Bugfix Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: test-requirements.txt [hunk 1]
```diff
@@ -8,4 +8,5 @@ coverage==4.5.4
 google-api-python-client
 grpcio
 protobuf
-git+https://github.com/ansible-community/pytest-ansible-units.git
+pytest-ansible ; python_version >= '3.9'
+git+https://github.com/ansible-community/pytest-ansible-units.git ; python_version < '3.9'
```
Comment (KB-perByte) [inline test-requirements.txt:12]: We do not support anything less than 3.9, why do we need this?

File: test-requirements.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
-black==22.3.0 ; python_version > '3.5'
+black==23.3.0 ; python_version > '3.7'
```
Comment (KB-perByte) [inline test-requirements.txt:1]: Same as below

File: test-requirements.txt [hunk 3]
```diff
@@ -1,4 +1,4 @@
-black==22.3.0 ; python_version > '3.5'
+black==23.3.0 ; python_version > '3.9'
```
Comment (NilashishC) [inline test-requirements.txt:1]: I don't know if we really need the version checks here. The min we support is 3.9 anyway and it's going to be 3.10 after May.

File: test-requirements.txt [hunk 4]
```diff
@@ -8,4 +8,4 @@ coverage==4.5.4
 google-api-python-client
 grpcio
 protobuf
-git+https://github.com/ansible-community/pytest-ansible-units.git
+pytest-ansible ; python_version >= '3.9'
```
Comment (NilashishC) [inline test-requirements.txt:11]: Same as above.

Comment (NilashishC) [reply test-requirements.txt:11]: I think you also need `pytest-xdist` here

**Pull request merged**